src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2120,9 +2120,9 @@ internal AssemblyName GetNameOfAssemblyReferencedByNode(NodeModel node)
         {
             AssemblyName assemblyName = null;
             // Get zerotouch assembly
-            if (node is DSFunction)
+            if (node is DSFunction || node is DSVarArgFunction)
             {
-                var descriptor = (node as DSFunction).Controller.Definition;
+                var descriptor = node is DSFunction function ? function.Controller.Definition :  (node as DSVarArgFunction).Controller.Definition;
```

Comment:
I think you can get rid of all these cases and the if statement above - just check if the type is `DSFunctionBase` - it has a `Controller.Definition` property you can access and you don't need to check any types, it's safer an faster - unless you need special behavior for each type.

Reply:
Checking if is DSFunctionBase type I've simplified the code in the next way:
![image](https://user-images.githubusercontent.com/61755417/216139960-fe6acdeb-6edc-4fca-a803-84c3010424fd.png)
Please let me know if it sounds good to you. Thanks
commit: https://github.com/DynamoDS/Dynamo/pull/13703/commits/f00db5b919450b0ccdafac4c9f90a8827b5d80c1